test/kubernetes_test.go

```diff
@@ -28,189 +26,256 @@ var dnsTestCases = []test.Case{
 		Qname: "bogusservice.test-1.svc.cluster.local.", Qtype: dns.TypeA,
 		Rcode:  dns.RcodeNameError,
 		Answer: []dns.RR{},
+		Ns: []dns.RR{
+			test.SOA("cluster.local.	300	IN	SOA	ns.dns.cluster.local. hostmaster.cluster.local. 1502313310 7200 1800 86400 60"),
+		},
 	},
 	{
 		Qname: "bogusendpoint.svc-1-a.test-1.svc.cluster.local.", Qtype: dns.TypeA,
```

Comment:
Can we also add a little comment above each of these of *what* it actually tests?